Kick-Starter/Kick-Starter/Display Coaches/DisplayCoachesTableViewCell.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  DisplayCoachesTableViewCell.swift
+//  Kick-Starter
+//
+//  Created by Abdullah Nana on 2021/11/16.
+//
+
+import UIKit
+
+class DisplayCoachesTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var coachNameLabel: UILabel!
+    @IBOutlet weak var coachPhotoImageView: UIImageView!
+    @IBOutlet weak var coachNationalityLabel: UILabel!
+    @IBOutlet weak var coachAgeLabel: UILabel!
+    @IBOutlet weak var imageStackView: UIStackView!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        applyCellStyling()
+    }
+    
+    private func applyCellStyling() {
+        backgroundColor = .clearBackgroundColor
+        selectionStyle = .none
+        
+        imageStackView.layer.cornerRadius = 20
```

Comment:
So even stuff like this can live in a style kit, that way all your cells where you display an image can have the same corner radius 😄